x/wasm-storage/keeper/abci.go

```diff
@@ -149,15 +161,26 @@ func (k Keeper) ProcessTallies(ctx sdk.Context) error {
 			"request_id", req.ID,
 			"post_result", postRes,
 		)
+		ctx.EventManager().EmitEvent(
+			sdk.NewEvent(
+				types.EventTypeTallyCompletion,
+				sdk.NewAttribute(types.AttributeRequestID, req.ID),
+				sdk.NewAttribute(types.AttributeTypeConsensus, strconv.FormatBool(consensus)),
+				sdk.NewAttribute(types.AttributeTallyVMStdOut, strings.Join(vmRes.Stdout, "\n")),
+				sdk.NewAttribute(types.AttributeTallyVMStdErr, strings.Join(vmRes.Stderr, "\n")),
+			),
+		)
```

Comment:
Small favour, can we add the `exit_code` attribute here which has the exit code of the tally module?

Reply:
done!